README.md

```diff
@@ -100,11 +100,16 @@ Aleth supports EVMC out of the box, but geth not yet.
 
 1. Build geth with EVMC
 
-	Checkout the `evmc` branch of [this geth fork](https://github.com/chfast/go-ethereum/tree/evmc) ([PR](https://github.com/ethereum/go-ethereum/pull/17050)):
+	Checkout the `ewasm` branch of [the ewasm geth fork](https://github.com/ewasm/go-ethereum):
```

Comment:
No need to state `ewasm` branch. That is the master, just say clone the ewasm geth fork.